Docs/reference/content/reference/driver/crud/reading.md

```diff
@@ -312,4 +312,122 @@ await OutAsync("myNewCollection");
 ```
 ```json
 { $out: 'myNewCollection' }
-```
\ No newline at end of file
+```
+
+### Text Search
+[Atlas Search](https://docs.atlas.mongodb.com/atlas-search) makes it easy to build fast, relevance-based search capabilities on top of your MongoDB data. Try it today on [MongoDB Atlas](https://www.mongodb.com/cloud/atlas), our fully managed database as a service.
+
+The following example can be used alongside the [Atlas Search Tutorial](https://docs.atlas.mongodb.com/reference/atlas-search/tutorial/)
+
+```csharp
+var client = new MongoClient(connectionString);
+var collection = client.GetDatabase("sample_mflix").GetCollection<BsonDocument>("movies");
+var searchStage = BsonDocument.Parse(@"
+{
+	$search: {
+    	'text': {
+    	    'query': 'baseball',
+            'path': 'plot'
+        }
+    }
+}");
+PipelineDefinition<BsonDocument, BsonDocument> pipeline = new []
+{
+    searchStage,
+    new BsonDocument("$limit", 5),
+    new BsonDocument
+    {
+        {"$project", new BsonDocument{ {"_id", 0}, {"title", 1}, {"plot", 1} }}
+    }
+};
```

Comment:
I find it a bit odd that:

1. the search stage is defined separately but the the next two stages are not
2. the search stage is created by parsing a BsonDocument but the next two stages are not

Reply:
For an example in documentation I would be consistent about creating all the stages the same way.

Something like this:

    var pipeline = new string[]
    {
        @"
        {
            $search :
            {
                text : 
                {
                    query : 'baseball',
                    path : 'plot'
                }
             }
        }",
        @"
        {
            $limit: 5
        }",
        @"
        {
            $project:
            {
                _id : 0,
                title : 1,
                plot : 1
            }
        }"
    }
    .Select(s => BsonDocument.Parse(s))
    .ToList();

    var moviesAboutBaseball = collection.Aggregate<BsonDocument>(pipeline).ToList();

We can quibble about whether to use shell formatting conventions or C# driver formatting conventions in the JSON strings...